recipes/paho-mqtt-c/all/patches/0002-fix-MinGW-and-OSX-builds-for-1-3-5.patch

```diff
@@ -1,8 +1,17 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index b53745b..d8787d8 100644
+index 531c39c..a1fd06b 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -41,7 +41,7 @@
+@@ -17,7 +17,7 @@
+ 
+ # Note: on OS X you should install XCode and the associated command-line tools
+ 
+-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.12)
++CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
```

Comment:
This change wasn't necessary

Our hook complains when the CMake minimum version is < 3.1 in the test_package. The minimum version everywhere else won't be checked